administrator/components/com_media/Controller/Api.php

```diff
@@ -104,7 +105,11 @@ public function files()
 			switch ($method)
 			{
 				case 'get':
-					$data = $this->getModel()->getFiles($adapter, $path, $this->input->getWord('filter'));
+					$options = array();
+					$options[] = $this->input->getWord('opt');
```

Comment:
I would make it an indexed array, where the key is here url and the value true or false.